benchmarks/src/btreemap.rs

```diff
@@ -219,6 +220,34 @@ pub fn btreemap_insert_10mib_values() -> BenchResult {
     })
 }
 
+#[bench]
+pub fn btreemap_iter_count() -> BenchResult {
```

Comment:
I'd also be curious how performance is like when the value is very small (e.g. a `u8`). Also, ideally, the benchmarks are added in one PR, and in a follow-up PR we make the improvements to the benchmarks.

Reply:
The benchmarks are now in separate [PR](https://github.com/dfinity/stable-structures/pull/191), and new one is added.